packages/shorebird_cli/lib/src/artifact_manager.dart

```diff
@@ -347,6 +347,21 @@ class ArtifactManager {
         .firstWhereOrNull((directory) => directory.path.endsWith('.app'));
   }
 
+  /// Returns the build/ subdirectory containing the compiled Windows exe.
+  Directory getWindowsReleaseDirectory() {
+    final projectRoot = shorebirdEnv.getShorebirdProjectRoot()!;
+    return Directory(
+      p.join(
+        projectRoot.path,
+        'build',
+        'windows',
+        'x64',
```

Comment:
Eventually this method may need to support arch, since I think windows does support arm64. 🤷

Reply:
Yep. When we add support for arm64 :)